x-pack/plugins/maps/public/embeddable/README.md

```diff
@@ -1,143 +0,0 @@
-
-### Map specific `input` parameters
```

Comment:
Removing this readme. Its pretty out dated and not really needed now that map embeddable is typed.